src/main/kotlin/no/nav/amt/deltaker/bff/deltakerliste/tiltakstype/Tiltakstype.kt

```diff
@@ -1,15 +1,42 @@
 package no.nav.amt.deltaker.bff.deltakerliste.tiltakstype
 
 import no.nav.amt.deltaker.bff.deltaker.model.Innhold
-import no.nav.amt.deltaker.bff.deltakerliste.Tiltak
+import no.nav.amt.deltaker.bff.deltaker.model.Innsatsgruppe
 import java.util.UUID
 
 data class Tiltakstype(
     val id: UUID,
     val navn: String,
-    val type: Tiltak.Type,
+    val tiltakskode: Tiltakskode,
+    val arenaKode: ArenaKode,
```

Comment:
Jeg valgte å ikke gjøre `arenaKode` til nullable her, vi vet at det finnes koder på de tiltakene vi jobber med, så det er litt ekstra jobb å innføre null-sjekker på de, samt at vi kanskje ikke ønsker å ha de med videre.

Reply:
Enig. Etterhvert ser jeg for meg at vi slutter å bruke arenakodene i hele verdikjeden, og da kan de jo bare fjernes :)